sklearn/cluster/hierarchical.py

```diff
@@ -15,7 +15,7 @@
 from scipy.sparse.csgraph import connected_components
 
 from ..base import BaseEstimator, ClusterMixin
-from ..externals.joblib import Memory
+from joblib import Memory
```

Comment:
I don't think this needs importing if all references to it in code have been removed...?